### PR TITLE
Multiline chat input

### PR DESCRIPTION
**Pull request by alwx**
I've copied some code from drawer to chat to allow users to send messages by tapping Enter on the keyboard (previously it didn't work well on iOS)

### CODE REVIEW

**Comment by rasom** [comment]
This doesn't look like correct behavior

b

File: src/status_im/chat/views/message_input.cljs [hunk 1]
```diff
@@ -54,9 +51,16 @@
                  :accessibility-label    :input
                  :on-focus               #(do (dispatch [:set :focused true])
                                               (dispatch [:set-chat-ui-props :show-emoji? false]))
-                 :on-blur                #(do (dispatch [:set :focused false])
+                 :on-blur                #(do (reset! message-text "")
```
Comment (rasom) [inline src/status_im/chat/views/message_input.cljs]: > WARNING: Use of undeclared Var status-im.chat.views.message-input/message-text at line 54 src/status_im/chat/views/message_input.cljs

**Comment by alwx** [comment]
Sounds crazy, but it's almost 2017 and React Native still doesn't completely support the desired behaviour: https://github.com/facebook/react-native/issues/1882
However I've found the better solution and also adapted it for drawer input.

**Comment by rasom** [comment]
@alwx, maybe I just don't understand, but why should we send the message when user press Return? I imagine that users will expect a new line in the result of this action. Or I miss something?

**Comment by jarradh** [comment]
hmm on return it shouldn't send the message

**Comment by alwx** [comment]
Ok, sorry. I will close the PR.

**Pull request closed without merging**